### DISCUSSION TITLE
Whats wrong with pyright LSP in nvim: cannot resolve import

### DISCUSSION THREAD

**Discussion by SdKay** [microsoft/pyright#8125] [General]
- All python file in the same folder
- `Contex` in `context.p`y import could be resolved successfully, but `Manager` in `manage.py` failed
- I tried use relative path or config `pyproject.toml`

![eSearch-2024-06-12-11-22-39-553](https://github.com/microsoft/pyright/assets/18526517/31896723-e47f-43ba-901b-509ea22f4b5f)

**Comment by SdKay** [comment]
I reboot nvim and problem gone. both `from .manager` and `from manager` is ok.
`pyproject.toml` make `from manager import Manager` work and its need nvim reboot for reloading pyright.
But why `from .manager import Manager` not working before I reboot nvim?

**Comment by erictraut** [comment]
Converting to a discussion since this is probably not a bug in pyright. You may need to reach out to someone familiar with the LSP support in nvim to answer your question.
